XCTest3/XCTestCase.swift

```diff
@@ -16,68 +16,8 @@
 
 import XCTest
 import Foundation
-import Boilerplate
 
 #if swift(>=3.0)
-    #if os(Linux)
```

Comment:
Not needed. They will not be realized in distant future. So If someone want to create tests for Linux, they must disable tests with this methods. It's better than crash in runtime